tests/consumers/acero_consumer.py

```diff
@@ -0,0 +1,52 @@
+import pyarrow as pa
+from pyarrow.lib import Table
+
+from .consumer import AbstractConsumer
+
+try:
+    import pyarrow.substrait as substrait
+except ImportError:
+    substrait = None
+
+
+class AceroConsumer(AbstractConsumer):
+    """
+    Implementation of the acero substrait consumer Class for testing
+    """
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def run_substrait_query(substrait_query: str) -> type[Table]:
+        """
+        Run the given substrait query and return the result
+
+        Args:
+            substrait_query: A json formatted string representing the
+            substrait query plan
+
+        Returns:
+            A pyarrow table resulting from running the substrait query plan.
+        """
+        buf = pa._substrait._parse_json_plan(substrait_query)
```

Comment:
This looks like a private module -- is there a public version of this you can hit?

Reply:
And if there isn't, can `pyarrow` add one?  You shouldn't be required to rely on a private API to do this (to my mind)

Reply:
@vibhatha is there a public version of this?  I couldn't find one.

Reply:
@richtia there was one I wrote for the initial PR, but we sort of made it private. The reason was we never wanted to expose it as a public feature at the time of the public API. But this doesn't restrict us from adding one as a utility. 
cc @westonpace shall we create a util API for Substrait in Python including this feature. I can create a quick PR for this if this is okay.